src/WebtoolsGeocoding.php

```diff
@@ -22,7 +24,22 @@ class WebtoolsGeocoding extends AbstractHttpProvider {
   /**
    * @var string
    */
-  const ENDPOINT_URL = 'http://europa.eu/webtools/rest/geocoding/?f=json&text=%s&maxLocations=%d&outFields=*';
+  const ENDPOINT_URL = 'https://europa.eu/webtools/rest/geocoding/?f=json&text=%s&maxLocations=%d&outFields=*';
+
+  /**
+   * Optional referer.
+   *
+   * @var string
+   */
+  private $referer;
+
+  /**
+   * Constructs a WebtoolsGeocoding provider.
+   */
+  public function __construct(HttpClient $client, string $referer = '') {
```

Comment:
When something is missed, from a semantically angle, it makes more sense to default to `NULL`.

Reply:
Fine for me, the reason I chose this pattern is because it is consistent with the Nominatim provider which also has an optional referer header (ref. https://github.com/geocoder-php/nominatim-provider/blame/e49f1cf4fb3862e0089de4d9fb65d190d6293d7c/Nominatim.php#L57).

I agree that `NULL` is more appropriate.